### DISCUSSION TITLE
Question about impact of "Bump @hyas/doks from 0.3.5 to 0.4.0"

### DISCUSSION THREAD

**Discussion by mhite** [thuliteio/doks#663] [Q&A] [answered]
Hello. I noticed the layout got a little whacky after merging 0.3.5 -> 0.4.0 @hyaks/doks bump PR into my repo.

Any ideas on what might be causing it? Here are the screenshots of before and after:

Before:

![Screen Shot 2022-02-09 at 9 09 16 PM](https://user-images.githubusercontent.com/1464385/153344314-4ef5a5c2-e3ac-491a-9276-67123d6edfe7.png)

After:

![Screen Shot 2022-02-09 at 9 07 43 PM](https://user-images.githubusercontent.com/1464385/153344329-8874e2f2-e31a-4d31-8997-5012ebec1407.png)

You'll notice all the top nav got a bit wonky and every decided to shift left.

Thanks!

**Comment by h-enk** [comment, marked as answer]
If you could share a repo, that would be handier to help you out

**Reply by mhite** [reply]
@h-enk - Sure, I've shared it your GH account. Thanks! https://github.com/mhite/djctl-site

**Reply by dobaret** [reply]
Hello @h-enk, I'm having the exact same problem, I'd be interested in feedback. [https://github.com/dobaret/dobaret.github.io](https://github.com/dobaret/dobaret.github.io)

**Reply by h-enk** [reply]
@mhite, @dobaret, thanks for pointing this out. We're missing upgrade instructions here — I will add these from now on!

Upgrade instructions for upgrading `@hyas/doks` from 0.3.5 to 0.4.0:

- add `container-xxl` and `container-fluid` to `safelist` in `./config/postcss.config.js` (this will fix your issue)
- add `./node_modules/@hyas/doks/assets/scss/components/_alerts.scss` and `./node_modules/bootstrap/scss/_dropdown.scss` to `whitelister` in `./config/postcss.config.js`

**Reply by dobaret** [reply]
It worked, thank you very much! :)

**Reply by mhite** [reply]
@h-enk - Thank you! Things look back to normal!

**Reply by james-s-w-clark** [reply]
Can confirm it worked for me too!
